cedar-policy-validator/src/human_schema/test.rs

```diff
@@ -317,6 +317,46 @@ mod demo_tests {
         assert!(as_src.contains(expected), "src was:\n`{as_src}`");
     }
 
+    #[test]
+    fn context_is_common_type() {
+        assert!(SchemaFragment::from_str_natural(
```

Comment:
Could you add some non-empty contexts too for completeness? E.g., the `authcontext` example from the [original RFC](https://github.com/cedar-policy/rfcs/blob/main/text/0024-schema-syntax.md).

Reply:
Done.